src/bindings/python/BUILDING.md

```diff
@@ -206,17 +206,17 @@ Set up the OpenVINO™ environment in order to add a module path to `PYTHONPATH`
 
     %OPENVINO_BASEDIR%\openvino_dist\setupvars.bat
 
-### Build an nGraph Python Wheel on Windows
+### Build a Python Wheel on Windows
 
 Build the Python wheel package:
 
-    cd "%OPENVINO_BASEDIR%/openvino/ngraph/python"
+    cd "%OPENVINO_BASEDIR%/openvino/bindings/python"
     python setup.py bdist_wheel
 
-The final wheel should be located in the `ngraph\python\dist` directory.
+The final wheel should be located in the `bindings\python\dist` directory.
 
-    dir openvino\ngraph\python\dist\
-    10/09/2020  04:06 PM         4,010,943 ngraph_core-0.0.0-cp38-cp38-win_amd64.whl
+    dir openvino\bindings\python\dist\
+    10/09/2020  04:06 PM         4,010,943 openvino-0.0.0.dev0-cp38-cp38-win_amd64.whl
```

Comment:
do we really have 0.0.0 as a version? I suppose version is detected here https://github.com/openvinotoolkit/openvino/blob/master/cmake/developer_package/version.cmake even without `git`

Reply:
The 0.0.0.dev0 is supplied when the environment variable NGRAPH_VERSION is not set:

`__version__ = os.environ.get("NGRAPH_VERSION", "0.0.0.dev0")`
openvino/src/bindings/python/setup.py:19

The docs example does not provide the information to set the version, however, I can include it, if necessary.

Reply:
Thank you for bringing attention to this problem, just now I have noticed that the path provided in the docs is still incorrect